### PR TITLE
[Merged by Bors] - chore: Golf `Finset.exists_subsuperset_card_eq`

### PR DESCRIPTION
**Pull request by YaelDillies**
Thanks Bhavik!

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Data/Finset/Card.lean
Comment (eric-wieser) [inline Mathlib/Data/Finset/Card.lean:626]: This bugged me, so I've made a start on https://github.com/leanprover-community/mathlib4/pull/14431

Comment (YaelDillies) [reply Mathlib/Data/Finset/Card.lean:626]: Nerdsniping Eric is the best kind of nerdsnipe 😝

**Review by rwst — APPROVED** [review]
LGTM

File: Mathlib/Data/Finset/Card.lean
Comment (b-mehta) [reply Mathlib/Data/Finset/Card.lean:626]: This was the best possible outcome of this golf :D

**Review by b-mehta — APPROVED** [review]
lgtm but I wrote it, so someone else should review too.

**Comment by eric-wieser** [comment]
I'd prefer to block this on #14431, to motivate review there once I finish up in an hour or so

**Review by eric-wieser — APPROVED** [review]
Nevermind, I can't actually make `decreasingInduction'` work with `induction` after all, it's not an induction principle in the usual sense because the inductive step depends on the original value.

bors merge

**Pull request closed without merging**
